x-pack/plugins/security_solution/public/common/components/events_viewer/index.tsx

```diff
@@ -122,6 +137,7 @@ const StatefulEventsViewerComponent: React.FC<Props> = ({
             kqlMode={kqlMode}
             query={query}
             onRuleChange={onRuleChange}
+            handleCloseExpandedEvent={handleCloseExpandedEvent}
```

Comment:
as you're passing `handleCloseExpandedEvent` already then there is no need to call it inside `EventsViewer` with `id` and `expandedEvent`
https://github.com/elastic/kibana/pull/83963/files#diff-49bac62091d5353d4aaabbf25de76f60b0b1227331326226fe7ec2efa4ed89c7R243